rad/src/lib.rs

```diff
@@ -267,11 +269,14 @@ pub fn run_aggregation_report(
     aggregate: &RADAggregate,
     settings: RadonScriptExecutionSettings,
     active_wips: ActiveWips,
-) -> Result<RadonReport<RadonTypes>> {
-    let context = &mut ReportContext::from_stage(Stage::Aggregation);
+) -> (Result<RadonReport<RadonTypes>>, ReportContext<RadonTypes>) {
```

Comment:
The documentation of this function has not been updated to mention that now it also returns a `ReportContext`. Same in `run_tally_report`.